rustainers/src/runner/podman.rs

```diff
@@ -35,6 +39,29 @@ impl InnerRunner for Podman {
         Cmd::new("podman")
     }
 
+    #[tracing::instrument(level = "info", skip(self), fields(runner = %self))]
+    fn is_inside_container(&self) -> bool {
+        Path::new("/run/.containerenv").exists()
+    }
+
+    #[tracing::instrument(level = "debug", skip(self), fields(runner = %self))]
+    async fn list_custom_networks(&self) -> Result<Vec<NetworkInfo>, ContainerError> {
+        let mut cmd: Cmd<'_> = self.command();
+        cmd.push_args(["network", "ls", "--no-trunc", "--format={{json .}}"]);
+        let mut result = cmd.json_stream::<NetworkInfo>().await?;
+        result.retain(|x| ["podman"].contains(&x.name.as_str()));
+        Ok(result)
+    }
+
+    #[tracing::instrument(level = "debug", skip(self), fields(runner = %self))]
+    async fn list_network_config(
+        &self,
+        network_id: ContainerId,
+    ) -> Result<Vec<IpamNetworkConfig>, ContainerError> {
+        let path = ".Subnets".to_string();
```

Comment:
🤌 **nitpick**: you do not need the allocation here.
This is fixed in my next commit